src/ts/ssh/sshSession.ts

```diff
@@ -332,6 +346,7 @@ export class SshSession implements Disposable {
 			// When encrypting, the key-exchange step will wait on the version-exchange.
 			// When not encrypting, it must be directly awaited.
 			await withCancellation(this.versionExchangePromise!, cancellation);
+			this.raiseReportProgress(Progress.ReceivingProtocolVersionExchange);
```

Comment:
Should it be "Completed" instead of "Receiving"? At this point after awaiting `versionExchangePromise`, the version exchange is completed.

And same below for the other case of ReceivingProtocolVersionExchange.

Reply:
Thanks for reviewing - even on holidays no less!

I had the same thought, was going to update. Will change that